django_python3_ldap/management/commands/ldap_clean_users.py

```diff
@@ -0,0 +1,69 @@
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+
+from django_python3_ldap import ldap
+from django_python3_ldap.conf import settings
+
+
+class Command(BaseCommand):
+
+    help = "Remove local user models for users not find anymore in the remote LDAP authentication server."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'purge',
+            action='store_true',
+            help='Purge instead of deactive local user models'
+        )
+
+    @staticmethod
+    def _remove(connection, user, purge, verbosity):
+        """
+        Deactivate or purge a given local user
+        """
+        if purge:
+            # Delete local user
+            try:
+                user.delete()
+                if verbosity >= 1:
+                    self.stdout.write("Purged {user}".format(
+                        user=user,
+                    ))
```

Comment:
Should catch a specific exception

Reply:
I am not sure to understand what you mean. I give a try with [6219174](https://github.com/etianen/django-python3-ldap/pull/240/commits/621917410672eebb33764518396980900ff9728d) commit